tests/test_type_aware.py

```diff
@@ -1103,6 +1103,99 @@ def testClientIDRead(self):
             self.assertNotEqual(data[column], value, column)
 
 
+class TestMariaDBTextTypeAwareDecryptionWithCiphertext(test_common.AcraCatchLogsMixin, test_common.BaseBinaryMariaDBTestCase,
+                                                       test_searchable_transparent_encryption.BaseTransparentEncryption):
+    # test table used for queries and data mapping into python types
+    test_table = sa.Table(
```

Comment:
I remember we had some problems with using shared metadata and registering tables with same names. And for that started to use `def get_encryptor_table(self)` methods and `test_common.SeparateMetadataMixin` mixin to use separate metadata if it can be used to avoid conflicts. please look at that, is it relevant here